code/game/objects/items/pinpointer.dm

```diff
@@ -11,18 +11,44 @@
 	item_state = "electronic"
 	throw_speed = 4
 	throw_range = 20
+	///What we're currently tracking
 	var/atom/movable/target
-	var/list/tracked_list
+	///The list of things we're tracking
+	var/list/tracked_list = list()
```

Comment:
Init list then immediately override it >:(

Reply:
Saddd

```suggestion
	///The list of things we're tracking
	var/list/tracked_list
```